README.md

```diff
@@ -2,13 +2,26 @@
 
 Currently we support python3 only.
 
+### Dependencies maintaince
+
+Since elastic beanstalk use `requirements.txt` to install packages by default and there is no way to change it.
+We can't use [`constraints.txt` concept](https://stackoverflow.com/questions/34645821/pip-constraints-files) directly for clear dependency organization.
+
+If you want to add new package to this project, follow the steps below:
+
+0. activate the virtual env
+1. `pip install xxx`
+2. add new item for that package in `requirements/base.txt` or `requirements/dev.txt`
+
+Normally, developer should use `pip install -r requirements/base.txt` on each fresh clone, which could install the latest compatible packages.
+
 ### Steps to run this app
 
 0. Create Environment variables like `QICHACHA_APPKEY` or `TENCENT_APPKEY`(or define them in `.env` file)
 
 ```sh
-pip install --user virtualenv
-pip install -r requirements.txt
+pip install --user virtualenv pre-commit
+pip install -r requirements/base.txt
```

Comment:
should _pip install -r requirements_ be called after env is created?

Reply:
my bad